embc-app/ClientApp/src/app/components/sidebox-add-organization/sidebox-add-organization.component.ts

```diff
@@ -12,11 +13,14 @@ export class SideboxAddOrganizationComponent implements OnInit {
 
   // TODO: THIS MUST BE IMPORTED AND MODIFIED BEFORE USE
   constructor(
-    private authService: AuthService
+    private authService: AuthService,
+    private uniqueKeyService: UniqueKeyService,
   ) { }
 
   ngOnInit() {
     this.authService.path.subscribe(p => {
+      // this is a "new" organization
+      this.uniqueKeyService.clearKey();
```

Comment:
This will clear the key when the component is displayed, which is probably incorrect.
Instead, should clear key when New button is clicked.